src/libmongoc/doc/mongoc_server_description_ismaster.rst

```diff
@@ -29,7 +29,7 @@ Parameters
 Description
 -----------
 
-The client or client pool periodically runs an `"isMaster" <https://docs.mongodb.org/manual/reference/command/isMaster/>`_ command on each server, to update its view of the MongoDB deployment. Use :symbol:`mongoc_client_get_server_descriptions()` and ``mongoc_server_description_ismaster()`` to get the most recent "isMaster" response.
+The client or client pool periodically runs a `"hello" <https://docs.mongodb.org/manual/reference/command/isMaster/>`_ command on each server, to update its view of the MongoDB deployment. Use :symbol:`mongoc_client_get_server_descriptions()` and ``mongoc_server_description_hello()`` to get the most recent "hello" response.
```

Comment:
The link to the docs should be updated to https://docs.mongodb.com/manual/reference/command/hello/.

Reply:
Updated